src/Microsoft.AspNet.SignalR.Core/Hubs/HubDispatcher.cs

```diff
@@ -549,6 +549,55 @@ private Task ProcessResponse(StateChangeTracker tracker, object result, HubReque
             return Transport.Send(hubResult);
         }
 
+        // This method is being used to buld a callback with reflection
+        private static void ContinueWith<T>(Task<T> task, DispatchingTaskCompletionSource<object> tcs)
```

Comment:
I removed this code in the previous checkin because it was unreferenced. Turned out that we use reflection and this is not dead code. Scary thing is that none of the tests run as part of build.cmd failed. I only noticed it because I ran functional tests I just fixed.